src/v2/guide/list.md

```diff
@@ -489,11 +489,14 @@ Here's a complete example of a simple todo list:
 
 ``` html
 <div id="todo-list-example">
-  <input
-    v-model="newTodoText"
-    v-on:keyup.enter="addNewTodo"
-    placeholder="Add a todo"
-  >
+  <form v-on:submit.prevent="addNewTodo">
+    <input
+      aria-label="Write a new todo item"
+      placeholder="E.g. Feed the cat"
+      v-model="newTodoText"
```

Comment:
I believe that we should keep the v-model as the first attribute, as that's what we're teaching here and it reduces the need for scanning on the part of the reader trying to learn.